goverge/main.py

```diff
@@ -133,6 +132,14 @@ def _parse_args(argv):
     )
 
     p.add_argument(
+        '--custom',
```

Comment:
Yeah that's a good point I wasn't really sure what to call it I will change that quick, @tomdeering-wf since I think you are already using this in your PR you will have to update your branch with the new flag name.